include/variables.h

```diff
@@ -168,10 +168,12 @@ extern u8 gSampleBankTable[];
 
 extern SaveContext gSaveContext;
 extern RegEditor* gRegEditor;
-extern u16 D_8015FCC0;
-extern u16 D_8015FCC2;
-extern u16 D_8015FCC4;
-extern u8 D_8015FCC8;
+
+extern u16 gCamAtSplinePointsAppliedFrame;
+extern u16 gCamEyeAppliedFrame;
+extern u16 gCamAtAppliedFrame;
```

Comment:
Maybe
```suggestion
extern u16 gCamEyeSetAppliedFrame;
extern u16 gCamAtSetAppliedFrame;
```
as well as `CS_CMD_CAM_EYE_SET` ?

Sorry I keep coming back to this lol, I'm just not a fan of this unused (and nearly useless) feature being named in such a generic fashion, e.g. `gCamAtSplinePointsAppliedFrame` contains `gCamAtAppliedFrame` so `gCamAtSplinePointsAppliedFrame` sounds like an extension of `gCamAtAppliedFrame`

Reply:
this reads as redundant to me, applied has the same meaning as set. its the frame it was set or the applied frame, it cant be both

Reply:
How should we refer to the "non-spline" at/eye thing?

Reply:
the fact that it lacks spline is enough to say that its not the spline

Reply:
since the spline ones have SplinePoints, I can add the word Point for parity, but beyond that, I think it maps well to CS_CAM_AT and CS_CAM_EYE and doesnt need any further terms added to it

Reply:
```suggestion
extern u16 gCamEyePointAppliedFrame;
extern u16 gCamAtPointAppliedFrame;
```
?

Reply:
>  I think it maps well to CS_CAM_AT and CS_CAM_EYE

well yeah I'm also suggesting changing those since they sound too generic, slap a "_set" in there or something that makes it clear it's a `eye = ` operation

Reply:
all of the other commands dont have "set" (CS_TIME, CS_LIGHT_SETTING etc)
and imo its not really needed

Reply:
the other commands don't have two versions for setting data

again I don't care if it's "set", just something that shows changing at/eye is split between two possibilities: spline and direct `=`

https://youtu.be/aUpJ3Ct2V-Q?t=84

Reply:
the lack of the term spline here is enough for me, and I want it to be phrased like the other commands. I really do think this is fine.
Ive added "point" to the global variables which alleviates the issue of it being confused with the other

Reply:
> all of the other commands dont have "set" (CS_TIME, CS_LIGHT_SETTING etc) and imo its not really needed

Actually there's a slight inconsistency here which may be why we're arguing

`CS_CMD_MISC` / `CutsceneCmd_Misc` (for each entry, not the whole list)
`CS_CMD_LIGHT_SETTING` / `CutsceneCmd_SetLightSetting` (for each entry, not the whole list)
`CS_CMD_START_SEQ` / `CutsceneCmd_StartSequence` (for each entry, not the whole list)
... (for each entry, not the whole list)
`CS_CMD_TIME` / `CutsceneCmd_SetTime` (for each entry, not the whole list)
`CS_CMD_CAM_EYE_SPLINE` / `CutsceneCmd_UpdateCamEyeSpline` (whole list)
`CS_CMD_CAM_AT_SPLINE` / `CutsceneCmd_UpdateCamAtSpline` (whole list)
`CS_CMD_CAM_EYE` / `CutsceneCmd_SetCamEye` (single element)
`CS_CMD_CAM_AT` / `CutsceneCmd_SetCamAt` (single element)
`CS_CMD_DESTINATION` / `CutsceneCmd_Destination` (single element)
`CS_CMD_TEXT` / `CutsceneCmd_Text` (for each entry, not the whole list)
`CS_CMD_TRANSITION` / `CutsceneCmd_Transition` (single element)

Reply:
yea i put set in the function names cause its the thing thats actually doing the operation. i still think its fine for the macro names to not have them

Reply:
what about `CutsceneCmd_Text` and such not having "set" ?

Reply:
you dont "set a text" or "set a misc", it just is the thing 
Destination, there is nothing to "set", destination is not a prexisting thing in the engine that youre setting like time. Its a higher level term for setting up a scene change

and transition is not really setting a transition, its a higher level name given to screen fills. this one can be argued to be changed to "color fill" really, but its higher level purpose is transitions between cuts in the same scene